nixos/modules/programs/environment.nix

```diff
@@ -30,7 +32,7 @@ in
       ];
 
     # TODO: move most of these elsewhere
-    environment.profileRelativeEnvVars =
+    environment.profileRelativeSessionVariables =
```

Comment:
Should we be using `environment.sessionVariables` for the assignments to `environment.variables` above as well?

Reply:
I'm going to migrate all uses of `environment.variables` to `sessionVariables` where appropriate (in another PR).
Also, I think that would infinitely recurse.